common-lib/src/main/scala/com/gu/mediaservice/model/InternalThrallMessage.scala

```diff
@@ -0,0 +1,9 @@
+package com.gu.mediaservice.model
+
+import org.joda.time.DateTime
+
+sealed trait InternalThrallMessage extends ThrallMessage {
+
+}
+
+case class CreateMigrationIndexMessage(migrationStart: DateTime, gitHash: String) extends InternalThrallMessage
```

Comment:
What is `gitHash`? Might be nice to have a comment to explain?

Reply:
@tjsilver and I moved this code when we refactored/added to Alex's PR, which didn't bring over the GH comment thread about this see https://github.com/guardian/grid/pull/3324#discussion_r638597056

Nonetheless I'll add a code comment